src/nft/components/details/AssetDetails.tsx

```diff
@@ -287,9 +288,13 @@ export const AssetDetails = ({ asset, collection }: AssetDetailsProps) => {
   const Filter = useCallback(
     function ActivityFilter({ eventType }: { eventType: ActivityEventType }) {
       const isActive = activeFilters[eventType]
+      const isDarkMode = useIsDarkMode()
 
       return (
-        <FilterBox isActive={isActive} onClick={() => filtersDispatch({ eventType })}>
+        <FilterBox
+          backgroundColor={isActive ? (isDarkMode ? vars.color.gray500 : vars.color.gray200) : undefined}
```

Comment:
nit: you could pass themevars.color.backgroundInteractive here instead of undefined to keep themeing consistent

Reply:
done